### ISSUE TITLE
stránka se všema infografikama

### ISSUE THREAD

**Issue by crysman** [faktaoklimatu/web-cz#745]
use case: hledám u nás na webu srovnání klimatických dohod, a nemám způsob, jak to najít, nemáme tam:
a) ani vyhledávání (preferred)
b) ani celkový přehled všeho co máme

uvítal bych dokud nebude a) tak mít aspoň funkční variantu b) - představuju si to prostě jako souhrnnou stránku uplně všeho, vlastně stejnou jako třeba https://faktaoklimatu.cz/temata/klima/ akorát tam bude uplně všechno ze všech oblastí (infografiky + studie)

už se mi to děje po několikáté, je tam toho prostě už hodně, a je i pro mě obtížné se trefit do TÉMATA, abych našel, co potřebuju... myslím, že pro public je to ještě obtížnější, a že jsme už poměrně nepřístupní pro tento druh lidí - co se chtějí vrátit ke konkrétní věci u nás na webu, ale neví, jaxe to menuje...

**Comment by jankrcal** [comment]
Pro seznam všech stránek můžeš použít https://faktaoklimatu.cz/sitemap.xml ;)
(no, to je spíš vtip).

Za sebe bych byl pro napřít síly do nějakého jednoduchého javascriptového vyhledávání.

Ale možná je to vlastně jednoduchá úprava stránky téma, aby vyplivla vše, když není vybraný žádný label, která se dá naprogramovat řádově za hodinku. Stačilo by Ti současné třízení obsahu podle typu, tedy prvně všechny infografiky, pak všechny studie, pak datasety? Přijde mi to takhle trochu user unfriendly, ale vlastně mi to celkem dává smysl.

Crysmane, máš nějaký návrh na to, kam na hlavní stránku / jinam hodit odkaz na takový seznam všeho?

Disclaimer, Martin teď rozděluje web na kód a obsah, tak teď myslím není vhodná chvíle něco takovýho programovat.

Tak jako tak, Martine, co si o tom myslíš?

**Comment by mukrop** [comment]
Za mna to, co popisuje Crysman je rozumny nizkonakladovy fix. Bol by to samostatny stitok "vsetko" alebo tak, implementacia je velmi rychla. Ak to chcete urobit, tak mozte, ja teraz potrebujem urobit review na existujuce PR a dotiahnut to rozdelovanie kodu a obsahu.

Vyhladavanie je samozrejme chcene a lepsie riesenie, ale trochu viac prace. Ale ak by sa do toho niekomu chcelo pustit, ja budem len za.

**Comment by jankrcal** [comment]
crysmane, máš chuť to naprogramovat?
(preferoval bych to tak, já si to můžu dát na seznam, ale mám spoustu jiných věcí pro fakta, tak to nebude pro mě priorita)

**Comment by crysman** [comment]
no, můžu zkusit, ale potřeboval bych od @mgrabovsky nebo @mukrop něco jako seznam klíčových slov, kudy na to - abych zbytečně neobjevoval kolo... šlo by?

**Comment by jankrcal** [comment]
Já ti taky asi můžu dát pár odkazů do kódu (zítra).

**Comment by jankrcal** [comment]
Rychlý prohledávání kódu. Nejsem si úplně jistej, co je nejsnazší cesta. Ono to totiž celý generuje plugin jekyll-archives (https://github.com/jekyll/jekyll-archives/tree/master/docs).

Asi bych šel cestou, že uděláš nový štítek "vse" s nejakym specialnim priznakem [1] a pak musíš 
 1. buď tento štítek ručně přidat ke všemu obsahu (a dát si pozor, aby se tento speciálni štítek nevypisoval u žádných grafik [2]),
 2. nebo to obejít na stránce šablony ke štítkům [3] a rovnou tam tahat všechen obsah,
 3. nebo nějak donutit ten plugin [4], aby zvládl vyplivnout všechen obsah, aniž by tam byl ten speciální štítek nastrkanej. Ideálně nějakou standardní konfigurací nebo jakkoliv jinak bez zásahu do toho kódu, protože by nám už nešlo upgradovat (nevím, jestli jde).

Za mě nejlépe 3 (možná nejde), případně 2, nejhůř 1.

Novým štítkem se ti vytvoří stránka https://faktaoklimatu.cz/temata/vse.
Tak jako tak musíš upravit tu šablonu [3] pro jekyll-archives, aby se s kategorií vše dobře popasovala.

Odkazy do kódu:
 [1] konfigurace štítků: _data/tags.yml
 [2] include štítků: _includes/tags.html
 [3] šablona stránky obsahu pro daný štítek: _layouts/tag.html
 [4] plugin: _plugins/jekyll-archives/archive.rb

**Comment by mukrop** [comment]
No, vsetko su to tak trochu hot-fix riesenia. Alebo mozes pomoct nasadeniu
vyhladavania tym, ze urobis prvy krok: Resers, ake vyhladavacie
pluginy/riesenia pre Jekyll existuju, co kto reportuje, ake su skusenosti a
podobne. Na zaklade toho by sa potom vybralo riesenie, ktorym pojdeme.

>

**Comment by crysman** [comment]
mně přijde užitečné obojí, chtěl bych obě jak:
a) vyhledávání, tak
b) stránku všeho

**Comment by jankrcal** [comment]
Ad vyhledávání, bez velké rešerše jsem udělal prototyp zde: https://github.com/faktaoklimatu/web-cz/pull/752
